### PR TITLE
Avoid duplicate git clone

### PR DESCRIPTION
**Pull request by LongyuZhang**
- Only git clone when the value of ADOPTOPENJDK_REPO or ADOPTOPENJDK_BRANCH changed.

Close: #1113

Signed-off-by: Longyu <longyu.zhang@ibm.com>

### CODE REVIEW

**Comment by LongyuZhang** [comment]
reviewers: @llxia

**Comment by LongyuZhang** [comment]
Tested internally on ppcle.
Saved average around 18 seconds per build.

File: buildenv/jenkins/JenkinsfileBase [hunk 1]
```diff
@@ -23,18 +23,20 @@ def makeTest(testParam) {
 def setupEnv() {
 	// if ADOPTOPENJDK_REPO and ADOPTOPENJDK_BRANCH are provided, clean the workpsace and re-download it
 	if (params.ADOPTOPENJDK_REPO && params.ADOPTOPENJDK_BRANCH) {
-		cleanWs()
+		if ( !params.ADOPTOPENJDK_REPO.contains( 'AdoptOpenJDK/openjdk-tests.git') || params.ADOPTOPENJDK_BRANCH != "master") {
```
Comment (llxia) [inline buildenv/jenkins/JenkinsfileBase]: Could we also update the comment in line 24?

Comment (smlambert) [reply buildenv/jenkins/JenkinsfileBase]: and fix spelling workspace instead of workpsace

Comment (LongyuZhang) [reply buildenv/jenkins/JenkinsfileBase]: The comment in line 24 has beed updated, with fixing "workspace" spelling. Thanks.

**Pull request merged**